symplyphysics/laws/quantities/quantity_is_linear_density_times_length.py

```diff
@@ -0,0 +1,66 @@
+"""
+Extensive quantity is linear density times length
+=================================================
+
+An extensive quantity of interest can be obtained by multiplying the corresponding linear
+density by length.
+"""
+
+from sympy import Eq, Symbol as SymSymbol
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    validate_input,
+)
+from symplyphysics.core.dimensions import assert_equivalent_dimension
+
+extensive_quantity = SymSymbol("extensive_quantity")
+"""
+Extensive quantity
+
+Symbol:
+    X
+"""
+
+linear_density = SymSymbol("linear_density")
+r"""
+Intensive linear density
+
+Symbol:
+    :math:`\lambda_X`
```

Comment:
Here write in ascii symbols, so it can be read without greek letters knowledge.
Put latex form in `Latex:` section.

Reply:
Sure